kolibri/core/assets/src/api-resources/contentNode.js

```diff
@@ -87,6 +87,18 @@ class ContentNodeResource extends Resource {
     }
     return promise;
   }
+  getAllContentCollection(resourceIds = {}, getParams = {}) {
+    const filteredResourceIds = this.filterAndCheckResourceIds(resourceIds);
+    let collection;
+    const key = this.cacheKey(getParams, { allContent: true }, filteredResourceIds);
```

Comment:
Create a unique cache key so that this collection can be retrieved again with caching.